backend/de.metas.acct.base/src/main/java-legacy/org/compiere/acct/Doc_Inventory.java

```diff
@@ -127,6 +132,7 @@ private void createFactsForInventoryLine(final Fact fact, final DocLine_Inventor
 				.setAmtSourceDrOrCr(costs.toMoney())
 				.setQty(line.getQty())
 				.locatorId(line.getM_Locator_ID())
+				.activityId(activityId)
```

Comment:
imho this would be necesary. Better check why org.compiere.acct.DocLine#getActivityId is not used/called?

Reply:
yup, was used if header activity took precedence over line activity.